presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -751,6 +751,30 @@ so a cross join between the two tables produces 125 rows::
     ...
     (125 rows)
 
+LATERAL
+^^^^^^^
+
+Subqueries appearing in the ``FROM`` clause can be preceded by the keyword ``LATERAL```.
+This allows them to reference columns provided by preceding FROM items.
+
+A ``LATERAL`` join can appear at the top level in the ``FROM`` list, or anywhere
+within a parenthesized join tree. In the latter case, it can also refer to any items
+that are on the left-hand side of a ``JOIN`` that it is on the right-hand side of.
+
+When a ``FROM`` item contains ``LATERAL`` cross-references, evaluation proceeds as follows:
+for each row of the FROM item providing the cross-referenced column(s),
+the ``LATERAL`` item is evaluated using that row or row set's values of the columns.
```

Comment:
This part is confusing to me:
> that row or row set's values of the columns

Where is there an "or"? Is this about singular vs plural or something else?

Reply:
> Is this about singular vs plural or something else?

Correct. I will remove the singular part.